src/components/SortBy.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div
+    :class="suit('')"
+    v-if="state"
+  >
+    <slot
+      :items="state.options"
+      :has-no-results="state.hasNoResults"
+      :refine="state.refine"
+      :current-refinement="state.currentRefinement"
```

Comment:
I would prefer to use computed properties rather than using `state` directly from the template. I found it more explicit when you look at the code. At the end it's the same but we should agree on one or the other and implement all the components in the same way.

Reply:
not really sure what the computed properties bring of value here :)

Reply:
Indeed, to me it's more explicit but it's only my opinion. At the end no matter the decision we choose but we should stick with one code style to avoid it will be easier to follow.